components/src/form-field/error.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from "@angular/core";
+
+import { I18nService } from "jslib-common/abstractions/i18n.service";
+
+// Increments for each instance of the input component
+let nextId = 0;
+
+@Component({
+  selector: "bit-error",
+  template: `<i class="bwi bwi-error"></i> {{ displayError }}`,
+  host: {
+    "[attr.id]": "id",
```

Comment:
The Angular style guide recommends using `@HostBinding` over this: https://angular.io/guide/styleguide#hostlistenerhostbinding-decorators-versus-host-metadata

>Why?
>The property associated with @[HostBinding](https://angular.io/api/core/HostBinding) or the method associated with @[HostListener](https://angular.io/api/core/HostListener) can be modified only in a single place —in the directive's class. If you use the host metadata property, you must modify both the property/method declaration in the directive's class and the metadata in the decorator associated with the directive.

We've used `@HostBinding` in the other components, so I think we should be consistent in that.